llvm-general-pure/src/LLVM/General/AST/FunctionAttribute.hs

```diff
@@ -37,7 +37,14 @@ data FunctionAttribute
     | StringAttribute {
         stringAttributeKind :: String,
         stringAttributeValue :: String -- ^ Use "" for no value -- the two are conflated
-      } 
+      }
+    | AllocSize Word64 (Maybe Word64)
```

Comment:
You've made these Word64, but they're "unsigned" in the C++ API. They should be Word.